File: JobApplicationTracker.Frontend/src/models/JobApplicationMinimal.ts
import {JAStatusType} from "./enums/JAStatusType";
import {JAEventType} from "./enums/JAEventType";

export interface JobApplicationMinimal {
    id: string,
    company: string,
    position: string,
    note: string | null,
    jaStatus: JAStatusType,
    jaEvent: JAEventType | null,
}

File: JobApplicationTracker.WPF/Models/JobApplicationMinimal.cs
using JobApplicationTracker.Wpf.Models.Enums;

namespace JobApplicationTracker.Wpf.Models;

public class JobApplicationMinimal
{
    public Guid Id { get; set; }
    public string Company { get; set; }
    public string Position { get; set; }
    public string? Note { get; set; }
    public JaStatusType JAStatus { get; set; }
    public JAEventMinimal? JAEvent { get; set; }
}